src/pymodaq_gui/managers/action_manager.py

```diff
@@ -82,8 +87,12 @@ def addaction(name: str = '', icon_name: str = '', tip='', checkable=False, chec
     enabled: bool
         set the enabled state
     """
+    
     if icon_name != '':
-        action = QAction(create_icon(icon_name), name, None)
+        if isinstance(icon_name,QtGui.QIcon):    
```

Comment:
no more needed!

Reply:
Same comment as above, how about a user who wants to use their own qicon